fastlane/lanes/release.rb

```diff
@@ -121,9 +205,52 @@
   #####################################################################################
   desc 'Prepare a new hotfix branch cut from the previous tag, and bump the version'
   lane :new_hotfix_release do |options|
-    hotfix_version = options[:version_name] || UI.input('Version number for the new hotfix?')
-    previous_tag = android_hotfix_prechecks(version_name: hotfix_version, skip_confirm: options[:skip_confirm])
-    android_bump_version_hotfix(previous_version_name: previous_tag, version_name: hotfix_version, version_code: options[:version_code])
+    new_version = options[:version_name] || UI.input('Version number for the new hotfix?')
+    new_version_code = options[:version_code] || UI.input('Version code for the new hotfix?')
+
+    ensure_git_status_clean
+
+    # Parse the provided version into an AppVersion object
+    parsed_version = VERSION_FORMATTER.parse(new_version)
+    previous_version = VERSION_FORMATTER.release_version(VERSION_CALCULATOR.previous_patch_version(version: parsed_version))
+
+    # Check versions
+    message = <<-MESSAGE
+
+      Current release version: #{current_release_version}
+      New hotfix version: #{new_version}
+
+      Current build code: #{current_build_code}
+      New build code: #{new_version_code}
+
+      Branching from tag: #{previous_version}
+
+    MESSAGE
+
+    if options[:skip_confirm]
+      UI.message(message)
```

Comment:
Similar to a previous comment, it looks like we can always print out the message and then conditionally do the confirmation.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/commit/6ab18efe9261a63b77766614c06c3bc661a697f6